src/main/webapp/partials/instructorHelpSessions.jsp

```diff
@@ -1,166 +1,92 @@
 <%@ page pageEncoding="UTF-8" %>
-<h4 class="text-color-primary" id="sessionTypes">Sessions</h4>
+<h2 class="text-color-primary" id="sessionTypes">Sessions</h2>
 <div id="contentHolder">
-  <br>
-  <ol style="list-style-type: none;">
+  <ol class="instructor-help-">
     <li id="fbSetupSession">
-      <span class="text-bold">
-          <b>1. Setting up a feedback session</b>
-        <br>
-      </span>
-      <div class="row">
-        <div class="col-lg-6">
-          To quickly create a session,
-          <p></p>
+      <h3>How do I set up a feedback session?</h3>
+      <div class="helpSectionContent">
+          To quickly set up a feedback session, follow the following steps:
           <ol>
-            <li>
-              Select a
-              <b>Session type</b>
-              <br>
-              <span class="text-muted">
-                <ul>
-                  <li>
-                    "session with my own questions"
-                  </li>
-                  <li>
-                    "session using template: team peer evaluation" (creates 5 standard questions)
-                  </li>
-                </ul>
-              </span>
-            </li>
-            <li>
-              Select a
-              <b>Course ID</b> for which the session will be created
-            </li>
-            <li>
-              Give it a nice
-              <b>Session name</b>
-            </li>
-            <li>
-              Select the
-              <b>Submission opening/closing time</b>
-              <span class="text-muted">
-                <ul>
-                  <li>
-                    This is the time period where a student can submit responses.
-                  </li>
-                </ul>
-              </span>
-            </li>
-            <li>
-              Click
-              <b>Create Feedback Session</b>!
-            </li>
+            <li><a href="#fbCreateSession">Create and schedule a new session</a></li>
```

Comment:
Here's the ordered lists are 1, 2, and 3
![image](https://user-images.githubusercontent.com/26651556/38189590-7591e938-3693-11e8-91ff-d023a65073ad.png)
but in the actual webpage these points are 2, 3 and 4. This may cause confusion.

Reply:
I can see how that might be problematic! Fixed in the latest commit (I used collapsibles instead of an ordered list to hold the questions) -- check out [this demo](https://teammates-cara.appspot.com/instructorHelp.jsp#sessionTypes).